apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -120,6 +120,11 @@ function AssessmentQuestionRow({
             : `${question.number_in_alternative_group}.`}
           ${question.title}
         </a>
+        ${question.manual_rubric_id == null
+          ? ''
+          : html`<span class="ml-2 text-info" title="This question uses a rubric">
```

Comment:
Did you consider using a Bootstrap tooltip here? I think that'll be a little friendlier to touch devices and just looks nicer than the rendered `title` attribute.

Reply:
Done in e03912c3f02eaa4d66318abe288693496eabba61.